src/components/Nurseries/TableCellRenderer.tsx

```diff
@@ -20,7 +20,7 @@ export default function NurseriesCellRenderer(props: RendererProps<TableRowType>
   }
 
   if (column.key === 'timeZone') {
-    return <CellRenderer index={index} column={column} value={value || defaultTimeZone.longName} row={row} />;
+    return <CellRenderer index={index} column={column} value={tz.get(value as string).longName} row={row} />;
```

Comment:
added this curried function / utility to get the timezone's long name (looks up value and falls back using default time zone behind the scenes)